docker-compose.yml

```diff
@@ -0,0 +1,42 @@
+version: '3'
+
+networks:
+  db-data-net:
+    driver: bridge
+
+volumes:
+ db-data:
+
+services:
+  mysql:
+    image: my-sql:1.2
+    build:
+      context: .
+      dockerfile: Dockerfile.mysql
+    container_name: my-sql
+    ports:
+     - "3306:3306"
+    volumes:
+     - db-data:/var/lib/mysql
+    environment:
+      - MYSQL_ROOT_PASSWORD=1234
```

Comment:
Using hardcoded passwords in the `docker-compose.yml` file is not recommended for security reasons. Consider using environment files or Docker secrets to manage sensitive information.